README.md

```diff
@@ -1986,16 +1987,35 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+## Line length
+
+  - [26.1](#26.1) <a name='26.1'></a> Avoid very long lines of code longer than 100 characters.
+
+  > Why? This ensures readability and maintainability.
+
+    ```javascript
+    // bad
+    var foo = { bar: "This is a bar.", baz: { "qux": "This is a qux" }, difficult: "to read" };
```

Comment:
this line should be obviously too long, as it is it's 96 characters, make it like 120+ chars